sdk/KoreBuild/modules/sharedsources/sharedsources.csproj

```diff
@@ -1,5 +1,13 @@
 <!-- this is a dummy project designed to produce contentFiles packages -->
-<Project Sdk="Microsoft.NET.Sdk">
+<Project>
+
+  <PropertyGroup>
+    <BaseOutputPath>$(NuspecBasePath)bin\</BaseOutputPath>
+    <BaseIntermediateOutputPath>$(NuspecBasePath)obj\</BaseIntermediateOutputPath>
+  </PropertyGroup>
+
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(NuspecBasePath), Directory.Build.props))\Directory.Build.props" Condition="Exists('$([MSBuild]::GetDirectoryNameOfFileAbove($(NuspecBasePath), Directory.Build.props))\Directory.Build.props')" />
```

Comment:
This `GetDirectoryNameOfFileAbove` sequence and the one for `Directory.Build.targets` could benefit from a descriptive property.